src/styles/variables.scss

```
@@ -0,0 +1,11 @@
+$grey-color: #616070;
+$black-color: #060b35;
+$blue-color: #00acdc;
+$white-color: #fff;
+
+$catalog-p-1: 50px;
```

```suggestion
$catalog-padding-top: 50px;
```
it's better to give more details in variables names